app/Platform/Actions/UpdatePlayerMetrics.php

```diff
@@ -11,16 +11,18 @@ class UpdatePlayerMetrics
 {
     public function execute(User $user): void
     {
-        $playerGames = $user->playerGames()->where('achievements_unlocked', '>', 0);
+        $playerGames = $user->playerGames()
+            ->join('GameData', 'GameData.ID', '=', 'player_games.game_id')
+            ->whereNotIn('GameData.ConsoleID', [100, 101]) // ignore events and hubs
+            ->where('achievements_unlocked', '>', 0);
         $user->achievements_unlocked = $playerGames->sum('achievements_unlocked');
         $user->achievements_unlocked_hardcore = $playerGames->sum('achievements_unlocked_hardcore');
         $user->completion_percentage_average = $playerGames->average('completion_percentage');
         $user->completion_percentage_average_hardcore = $playerGames->average('completion_percentage_hardcore');
 
-        // TODO refactor to use aggregated player_games metrics
-        $user->RAPoints = $user->achievements()->published()->wherePivotNotNull('unlocked_hardcore_at')->sum('Points');
-        $user->RASoftcorePoints = $user->achievements()->published()->wherePivotNull('unlocked_hardcore_at')->sum('Points');
-        $user->TrueRAPoints = $user->achievements()->published()->wherePivotNotNull('unlocked_hardcore_at')->sum('TrueRatio');
+        $user->RAPoints = $playerGames->sum('points_hardcore');
+        $user->RASoftcorePoints = $playerGames->sum('points') - $user->RAPoints;
+        $user->TrueRAPoints = $playerGames->sum('points_weighted');
```

Comment:
Will this PR follow in a later release? otherwise we'll have to wrap this and the replaced code in the aggregated queries feature flag

Reply:
I hadn't really considered the fact that these values are currently being used. This cannot be called until player_games is fully populated. I'll wrap it in the feature flag and restore the old code as the default path.